spec/factories/orders.rb

```diff
@@ -1,6 +1,11 @@
 FactoryGirl.define do
   factory :order do
-    
+    Status "Pending"
+    total   234344
```

Comment:
Separate every 3 digits in the integer portion of a number with underscores(_).<br>Put one space between the method name and the first argument.